bin/bundle

```diff
@@ -18,7 +18,7 @@ m = Module.new do
   end
 
   def env_var_version
-    ENV['BUNDLER_VERSION']
+    ENV.fetch('BUNDLER_VERSION', nil)
```

Comment:
Is this caused due to version difference?

Reply:
`fetch` allows for a default value if `BUNDLER_VERSION` isn't defined, which in this case is `nil`. This has the same behavior as `ENV['BUNDLER_VERSION']`.

Reply:
Rubocop made this change automatically